script/with-static.sh

```diff
@@ -4,6 +4,7 @@ set -ex
 
 export BUILD="$PWD/vendor/libgit2/build"
 export PCFILE="$BUILD/libgit2.pc"
+export PKG_CONFIG_PATH=$BUILD
```

Comment:
i believe you'll want to use the installed .pc file, not the one from the temporary build folder.